### PR TITLE
Create weekly-2020-11-27.md

### PR DESCRIPTION
**Pull request by andylokandy**
Create 2020-11-27

### CODE REVIEW

File: meetings/minutes/weekly-2020-11-27.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+* [Python client](https://github.com/andylokandy/client-py)
+  * Feature complete as rust client.
+
+* Keyspace
+  * Design document and github project: https://github.com/tikv/pd/issues/3214
+
+## Test & Bugfix
+
+* And fallpoint tests for `aapply_before_split()`: https://github.com/tikv/tikv/pull/9123
```
Comment (sticnarf) [inline meetings/minutes/weekly-2020-11-27.md:20]:
```suggestion
* And fallpoint tests for `apply_before_split()`: https://github.com/tikv/tikv/pull/9123
```

File: meetings/minutes/weekly-2020-11-27.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* Fix inconsistency caused by missing BatchGet response error: https://github.com/tikv/tikv/pull/9077
+* Fix async commit amend txn failure with update statement: https://github.com/pingcap/tidb/pull/21281
+* Fix IndexMergeReader works incorrectly under RC isolation level: https://github.com/pingcap/tidb/pull/21208
+* Fix async commit amend txn failure with update statement: https://github.com/pingcap/tidb/pull/21281
+
+## Others
+
+* Bug: batch_get doesn't return all results when encountering a memory lock: https://github.com/tikv/tikv/issues/9069
```
Comment (sticnarf) [inline meetings/minutes/weekly-2020-11-27.md:28]: This bug is included in last week's report.

Comment (sticnarf) [inline meetings/minutes/weekly-2020-11-27.md:28]: * Bug: Index inconsistency after add index when using async commit: https://github.com/pingcap/tidb/issues/21339

File: meetings/minutes/weekly-2020-11-27.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* Keyspace
+  * Design document and github project: https://github.com/tikv/pd/issues/3214
+
+## Test & Bugfix
+
+* And fallpoint tests for `aapply_before_split()`: https://github.com/tikv/tikv/pull/9123
+* Fix inconsistency caused by missing BatchGet response error: https://github.com/tikv/tikv/pull/9077
+* Fix async commit amend txn failure with update statement: https://github.com/pingcap/tidb/pull/21281
+* Fix IndexMergeReader works incorrectly under RC isolation level: https://github.com/pingcap/tidb/pull/21208
+* Fix async commit amend txn failure with update statement: https://github.com/pingcap/tidb/pull/21281
```
Comment (sticnarf) [inline meetings/minutes/weekly-2020-11-27.md:24]: duplicated item

Comment (andylokandy) [reply meetings/minutes/weekly-2020-11-27.md:24]: hummm 🤣

**Review by sticnarf — APPROVED** [review]

**Review by youjiali1995 — APPROVED** [review]

**Pull request merged**